consts/errors.go

```diff
@@ -43,4 +43,5 @@ var (
 	StatusServiceUnavailable = status.Error(codes.Unavailable, ErrServiceUnavailable.Error())
 	StatusNilRequestUser     = status.Error(codes.InvalidArgument, ErrNilRequestUser.Error())
 	StatusUUIDNotFound       = status.Error(codes.NotFound, ErrUUIDNotFound.Error())
+	StatusUUIDInvalid        = status.Error(codes.InvalidArgument, ErrInvalidUUID.Error())
```

Comment:
FYI, the convention of naming Errors should be prefixed by `Err`
https://golang.org/doc/effective_go.html#commentary